Dockerfile

```diff
@@ -20,8 +20,32 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -yqq && \
     python3-pylibacl \
     python3-pyxattr
 
+# Build dependencies specific for rdiff-backup development and testing
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -yqq && \
+    apt-get install -y --no-install-recommends \
+    tox \
+    rdiff \
+    # /usr/include/sys/acl.h is required by test builds
+    libacl1-dev \
+    # tox_slow uses rsync for comperative benchmarking
+    rsync
+
 # Build dev image
 # docker build --pull --tag rdiff-backup-dev:debian-sid .
 
-# Build rdiff-backup
-# docker run -it -v ${PWD}:/build -w /build rdiff-backup-dev:debian-sid ./setup.py build
+# Build rdiff-backup (assumes source is in directory 'rdiff-backup' and it's parent is writeable)
+# docker run -it -v ${PWD}/..:/build -w /build/rdiff-backup rdiff-backup-dev:debian-sid ./setup.py build
+
+RUN useradd -ms /bin/bash testuser
+
+# Run tests (note session will user=root inside Docker)
+# docker run -it -v ${PWD}/..:/build -w /build/rdiff-backup rdiff-backup-dev:debian-sid bash
+#   cd ..
+#   curl -LO https://github.com/ericzolf/rdiff-backup/releases/download/Testfiles2019-08-10/rdiff-backup_testfiles_2019-08-10.tar.gz
+#   tar xvf *.tar.gz # This must be run as root
+#   ./rdiff-backup_testfiles.fix.sh # This must be run as root
+#   cd rdiff-backup
+#   export RDIFF_TEST_UID=1000 # Tests require that there is a normal user
```

Comment:
it's an assumption that the testuser is created with ID 1000, probably safe in a container, but still an assumption. Don't you want to create the user with an explicit ID?

Also, wouldn't it make sense to set this environment variable as permanent environment variable in the container?